security/security-design/fn-datasafe-dbaudit-to-oci-logging/terraform/locals.tf

```diff
@@ -0,0 +1,32 @@
+###############################################################################
+# Copyright (c) 2022, 2023, Oracle and/or its affiliates.  All rights reserved.
+# This software is dual-licensed to you under the Universal Permissive License
+# (UPL) 1.0 as shown at https://oss.oracle.com/licenses/upl.
+###############################################################################
+#
+# Author: Fabrizio Zarri
+#
+################################################################################
+
+locals {
+  ocir_docker_repository = join("", [lower(lookup(data.oci_identity_regions.oci_regions.regions[0], "key")), ".ocir.io"])
+  ocir_namespace = lookup(data.oci_objectstorage_namespace.namespace, "namespace")
+  
+  namespace = lookup(data.oci_objectstorage_namespace.namespace, "namespace")
+  compartment_name = lookup(data.oci_identity_compartment.compartment, "name")
+  resource_nc = "-${var.deployment_name}-${var.region}-${var.purpose}-${random_id.tag.hex}"
+  policy_displayname = "${var.PolicyNamePrefix}${local.resource_nc}"
```

Comment:
Terraform naming convention is all lower case and words separated by '_'.

Reply:
I have update the name of all variables and references. Now are only present lower case and words separated by '_'.